### DISCUSSION TITLE
Best Resources to Learn Blockchain Technologies.

### DISCUSSION THREAD

**Discussion by akhileshthite** [EddieHubCommunity/support#1944] [Tips and Tricks]
All resources: https://www.notion.so/Getting-Started-with-Blockchain-and-Ethereum-92ba0d551e2f46a2adeed5d0e83fb3d0

**Comment by akhileshthite** [comment]
My blockchain tutorial: https://learn.figment.io/tutorials/build-a-social-media-dapp-and-deploy-it-on-polygon

**Reply by srisatya12** [reply]
Thanks for the info 👍🏻

**Reply by akhileshthite** [reply]
> Thanks for the info 👍🏻

Welcome :)

**Comment by vinzvinci** [comment]
I found this interesting, so I like to share here
https://dev.to/dabit3/the-new-creator-economy-daos-community-ownership-and-cryptoeconomics-lnl

**Comment by Sonichigo** [comment]
I too would like to share this:- https://www.postman.com/blockchainnow/workspace/tatum-io/overview

**Comment by KukretiShubham** [comment]
Thanks for sharing.
